pom.xml

```diff
@@ -32,7 +32,7 @@
         <exec.mainClass>org.rascalmpl.shell.RascalShell</exec.mainClass>
         <rascal.test.memory>3</rascal.test.memory>
         <maven.compiler.release>11</maven.compiler.release>
-        <rascal-maven.version>0.28.9-BOOT1</rascal-maven.version>
+        <rascal-maven.version>0.30.0-RC3</rascal-maven.version>
```

Comment:
I don't think this is part of this PR?

Reply:
yes these are essential to make then next step, at least in my mind. This is the first PR that will run the commandline versions of a previous version of rascal on the current version of rascal.